### PR TITLE
luci-app-adbyby-plus: fetch upstream from lean

### LINKED ISSUE #221: 广告屏蔽大师PLUS+问题
广告屏蔽大师PLUS+问题
高级设置中adblock Plus 数据库有问题，更新后规则7000以上，造成网络连接失败，比如拨号成功后无法打开网页，可否修复谢谢

### PR DESCRIPTION
**Pull request by XiaoliChan**
Fix #221 #123

### CODE REVIEW

**Comment by XiaoliChan** [comment]
Test with:
- x86

Confirm it works
image

File: applications/luci-app-adbyby-plus/luasrc/controller/adbyby.lua [hunk 1]
```diff
@@ -4,21 +4,16 @@ function index()
 	if not nixio.fs.access("/etc/config/adbyby") then
 		return
 	end
-
-	local page = entry({"admin", "services", "adbyby"}, alias("admin", "services", "adbyby", "base"), _("ADBYBY Plus +"), 9)
```
Comment (1715173329) [inline applications/luci-app-adbyby-plus/luasrc/controller/adbyby.lua:8]: keep it as-is

File: applications/luci-app-adbyby-plus/root/etc/init.d/adbyby [hunk 2]
```diff
@@ -3,13 +3,10 @@
 START=96
 STOP=10
 
-extra_command "add_rule"
```
Comment (1715173329) [inline applications/luci-app-adbyby-plus/root/etc/init.d/adbyby:6]: keep it as-is

File: applications/luci-app-adbyby-plus/root/etc/init.d/adbyby [hunk 3]
```diff
@@ -251,4 +238,4 @@ boot()
 {
 	mkdir -p /tmp/adbyby && cp -a /usr/share/adbyby /tmp/
 	start
-}
+}
```
Comment (1715173329) [inline applications/luci-app-adbyby-plus/root/etc/init.d/adbyby:241]: missing new line

File: applications/luci-app-adbyby-plus/root/usr/share/adbyby/ad-update [hunk 4]
```diff
@@ -1,11 +1,8 @@
...
-    cat /tmp/adnew.conf | grep ^\|\|[^\*]*\^$ | sed -e 's:||:address\=\/:' -e 's:\^:/0\.0\.0\.0:' > /tmp/ad.conf
-  fi
-fi
-rm -f /tmp/adnew.conf
+	if (grep -wq "address=" /tmp/adnew.conf) ; then
+		cp /tmp/adnew.conf /tmp/ad.conf
+	else
+		cat /tmp/adnew.conf | grep ^\|\|[^\*]*\^$ | sed -e 's:||:address\=\/:' -e 's:\^:/0\.0\.0\.0:' > /tmp/ad.conf
+	fi
+fi
```
Comment (1715173329) [inline applications/luci-app-adbyby-plus/root/usr/share/adbyby/ad-update:8]: missing new line

File: applications/luci-app-adbyby-plus/root/etc/uci-defaults/adbyby [hunk 5]
```diff
@@ -5,14 +5,12 @@ uci -q batch <<-EOF >/dev/null
 	add ucitrack adbyby
 	set ucitrack.@adbyby[-1].init=adbyby
 	commit ucitrack
-
```
Comment (1715173329) [inline applications/luci-app-adbyby-plus/root/etc/uci-defaults/adbyby:8]: why? linebreak makes it more clear

File: applications/luci-app-adbyby-plus/root/etc/uci-defaults/luci-adbyby [hunk 6]
```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```
Comment (1715173329) [inline applications/luci-app-adbyby-plus/root/etc/uci-defaults/luci-adbyby:1]: duplicate of the file above

**Review by 1715173329 — CHANGES_REQUESTED** [review]

File: applications/luci-app-adbyby-plus/root/etc/init.d/adbyby (hunk 2 above)
Comment (XiaoliChan) [reply applications/luci-app-adbyby-plus/root/etc/init.d/adbyby:6]: Why, it had new variable call `EXTRA_COMMANDS`

File: applications/luci-app-adbyby-plus/root/etc/uci-defaults/luci-adbyby (hunk 6 above)
Comment (XiaoliChan) [reply applications/luci-app-adbyby-plus/root/etc/uci-defaults/luci-adbyby:1]: Fixed!

File: applications/luci-app-adbyby-plus/root/etc/uci-defaults/adbyby (hunk 5 above)
Comment (XiaoliChan) [reply applications/luci-app-adbyby-plus/root/etc/uci-defaults/adbyby]: Fixed!

File: applications/luci-app-adbyby-plus/root/usr/share/adbyby/ad-update (hunk 4 above)
Comment (XiaoliChan) [reply applications/luci-app-adbyby-plus/root/usr/share/adbyby/ad-update]: Fixed!

File: applications/luci-app-adbyby-plus/root/etc/init.d/adbyby (hunk 3 above)
Comment (XiaoliChan) [reply applications/luci-app-adbyby-plus/root/etc/init.d/adbyby]: Fixed!

File: applications/luci-app-adbyby-plus/luasrc/controller/adbyby.lua (hunk 1 above)
Comment (XiaoliChan) [reply applications/luci-app-adbyby-plus/luasrc/controller/adbyby.lua:8]: Fixed!

File: applications/luci-app-adbyby-plus/root/etc/init.d/adbyby (hunk 2 above)
Comment (XiaoliChan) [reply applications/luci-app-adbyby-plus/root/etc/init.d/adbyby]: Fixed!

**Pull request merged**